internal/msg/errormsg.go

```diff
@@ -130,6 +130,8 @@ const (
 	InvalidEmulatorName = "missing `emulator` in emulator name: %s. Suite name: %s. Emulators index: %d"
 	// MissingEmulatorPlatformVersion indicates no emulator platform version provided
 	MissingEmulatorPlatformVersion = "missing platform versions for emulator: %s. Suite name: %s. Emulators index: %d"
+	// InvalidOtherAppsDescriptions indicates otherAppsDescriptions cannot match otherApps
+	InvalidOtherAppsDescriptions = "otherAppsDescriptions cannot match otherApps"
```

Comment:
I don't understand the error message 🤔 
Based on where it's used:
```
	if len(p.Espresso.OtherAppsDescriptions) > 0 && len(p.Espresso.OtherApps) != len(p.Espresso.OtherAppsDescriptions) {
		return errors.New(msg.InvalidOtherAppsDescriptions)
	}
```

I'd say the message you are trying to convey is "otherAppsDescriptions has fewer values than otherApps".

Message aside, I actually don't think you even need this check. It forces the user to provide all descriptions or none. I don't think we really care if the user provides just half of them, right?

Reply:
removed the `otherAppsDescriptions`